bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCghost.html

```diff
@@ -350,18 +350,16 @@
             <tr stype="padding-top:2%;"><td colspan="2"><b>Detector Binning:</b></td></tr>
             <tr stype="padding-top:2%;">
                <td colspan="1">
-                    Spectral Direction
-                    <select name="spectralBinning" size="1">
-                        <option value="ONE" selected>1</option>
-                        <option value="TWO">2</option>
-                        <option value="FOUR">4</option>
-                    </select>
-                   Spatial Direction
-                   <select name="spatialBinning" size="1">
-                       <option value="ONE" selected>1</option>
-                       <option value="TWO">2</option>
-                       <option value="FOUR">4</option>
-                       <option value="EIGHT">8</option>
+                   Binning
+                   <select name="binning" size="1">
```

Comment:
I changed the UI to use the same options as the OT

Reply:
When I started looking at and doing the merge with the scala part, I put in a document to talk to Andy, Bryan, etc, to resolve those conflicts. This is the link to the [document](https://docs.google.com/document/d/1FdDBVsxWxSugCjtlbcrs7wKgPjHCT6jDANVPNDRTqAw/edit#heading=h.jeok54id19fg) 

At that meeting, I think it was intended that the binning would be separate. What I was thinking at the time, was to get it separately and then have it coincide with Shane's.